tests/acceptance/qunit-dom-test.js

```diff
@@ -34,6 +34,9 @@ module('Acceptance | qunit-dom', function(hooks) {
     assert.dom('#hidden-input').isNotVisible();
     assert.dom('p').isVisible({ count: 2 });
 
+    let titleElement = document.querySelector('#title');
+    assert.dom(titleElement).isVisible();
```

Comment:
I don't think we need this test here. the tests here are essentially just smoke tests to see if the library is loaded correctly. there is no need to test every single feature here.

Reply:
Fair. I'll remove, since we cover `isVisible`/`isNotVisible` above adequately.